airbyte-webapp/src/components/connectorBuilder/Builder/StreamConfigView.tsx

```diff
@@ -49,7 +51,19 @@ export const StreamConfigView: React.FC<StreamConfigViewProps> = ({ streamNum })
       {/* Not using intl for the labels and tooltips in this component in order to keep maintainence simple */}
       <BuilderTitle path={streamFieldPath("name")} label="Stream Name" size="md" />
       <div className={styles.controls}>
-        <button className={styles.deleteButton} type="button" onClick={handleDelete}>
+        <AddStreamButton
+          onAddStream={(addedStreamNum) => {
+            setSelectedView(addedStreamNum);
+            setTestStreamIndex(addedStreamNum);
+          }}
+          initialValues={field.value[streamNum]}
+          button={
+            <button className={styles.controlButton} type="button" onClick={handleDelete}>
```

Comment:
This button should not have `onClick={handleDelete}`. Looks like it is being overwritten in AddStreamButton which I suppose is why this didn't cause the button to delete the stream, but it still shouldn't be here